docs/pages/more/expo-cli.mdx

```diff
@@ -461,43 +461,44 @@ From here, you can choose to generate basic project files like:
+| `EXPO_UNSTABLE_ATLAS`                  | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="experimental" /><StatusTag status="SDK 51+" /></div>Gather Metro bundle information during development or export.                                                                                                                                                                                                                                                  |
+| `EXPO_NO_BUNDLE_SPLITTING`             | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="experimental" /><StatusTag status="SDK 51+" /></div>Disable Metro splitting chunks on async imports in production (web-only).                                                                                                                                                                                                                                      |
+| `EXPO_USE_METRO_REQUIRE`               | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="SDK 52+" /></div>Enable the use of Expo's custom Metro `require` implementation and `string` based module IDs. This enables better debugging and deterministic IDs for React Server Components. Does not support legacy RAM bundles.                                                                                                                               |
+| `EXPO_UNSTABLE_METRO_OPTIMIZE_GRAPH`   | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="experimental" /><StatusTag status="SDK 52+" /></div>Enable eager bundling where transformation runs uncached after the entire bundle has been created. This is required for production tree shaking and is less optimized for development bundling.                                                                                                                |
+| `EXPO_UNSTABLE_TREE_SHAKING`           | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="experimental" /><StatusTag status="SDK 52+" /></div>Enable unstable tree shaking support across all platforms. See [tree shaking](/guides/tree-shaking) for more details.                                                                                                                                                                                          |
+| `EXPO_USE_DEPRECATED_POLYFILL_SORTING` | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="SDK 52" /></div>Enable the deprecated `serializer.getModulesRunBeforeMainModule` system which injects polyfill modules before running the main module. This has been removed in favor of simply importing the required polyfills in the `expo` module.                                                                                                             |
```

Comment:
```suggestion
| `EXPO_USE_DEPRECATED_POLYFILL_SORTING` | **boolean** | <div className="flex items-center pb-1.5"><StatusTag status="SDK 52" /></div>Enable the deprecated `serializer.getModulesRunBeforeMainModule` system which injects polyfill modules before running the main module. This has been removed in favor of simply importing the required polyfills in the `expo` package.                                                                                                             |
```

Small suggestion: we should use package instead of the module since we either prefer using package in the context of `expo` and for SDK libraries as per our writing style guide.